02-create.md

```diff
@@ -432,3 +432,37 @@ but it does find the copy in `thesis` that we didn't delete.
 > The command `ls -t` lists things by time of last change,
 > with most recently changed files or directories first.
 > In what order does `ls -R -t` display things?
+
+> ## Copy a folder structure sans files{.challenge}
+>
+> You're starting a new experiment, and would like to duplicate the file 
+> structure from your previous experiment without the data files so you can 
+> add new data. 
+>
+> Use the `file-structure` directory under `shell-data` as an example.
+> The goal is to copy the file structure of the `2016-5-18-data` folder
+> into a folder called `2016-5-20-data` and remove the data files from 
+> the first folder.
+> 
+> Navigate to the `file-structure` folder. Which of the following set of 
+> commands would achieve this objective? What would the other commands do?
+>
+>#1
```

Comment:
The `#1` gets rendered as a heading. See here: https://github.com/maglet/shell-novice/blob/f59bb3e9a3c05ed2a7f296f44f667cfe1835d718/02-create.md.

Just use `1.`, `2.`, and `3.` instead?